apps/src/netsim/NetSimLogBrowserFilters.jsx

```diff
@@ -159,7 +158,7 @@ export const SentByDropdown = React.createClass({
         <option value="none">
           {this.props.i18n.logBrowserHeader_sentByAnyone()}
         </option>
-        {_.uniq(this.props.logRows.map(row => row['sent-by']))
+        {this.props.senderNames
```

Comment:
An early suspicion was that the changing `logRows` prop was causing the dropdown to re-render, so I calculate the unique sender names outside of this component to make sure it's not interfering.  Actually I pull this all the way up to the `NetSimRouterLogModal` layer where it can take advantage of the incremental log updates, to minimize unneeded iteration over the full collection of logs.  This reduces work at the cost of increasing state; a tradeoff that seems appropriate for the log browser, which has been a performance hog in the past.